### ISSUE TITLE
Prevent gpx parse error with zip import ( global import fail)

### ISSUE THREAD

**Issue by paf38** [SamR1/FitTrackee#87]
Hi,

I discovered than some of my gpx files aren't parsed by gpxpy and when zip import is choosen, global import fail.

Is it possible to add with try/catch in zip import method and list invalid files at the end of the import on fittrackee_client ?

**Comment by sgofferj** [comment]
Yeah, same here! I manually do a quick check over all files before I import them. Most often I have problems because the files don't contain any actual trackpoints (stopped recording before GPS lock) or they are not properly closed.
My quick check is just deleting every file under 10k size and then runnning a tail -n2 over evey file so check if it's properly closed.

A big related problem is that fittrackee does not clean the import/unzip folders on failed import, so even after fixing erroneous files, the following import will also fail.
